HoloPipelines/core/adapters/nifti_file.py

```diff
@@ -78,11 +82,10 @@ def read_nifti_as_np_array(input_path: str, normalise: bool = True):
     :return:
     """
     nifti_image: nibabel.nifti1.Nifti1Image = read_nifti_image(input_path)
-
     if normalise:
-        nifti_image = normalise_nifti_image(nifti_image)
-
-    nifti_image_as_np_array: np.array = extract_np_array_from_nifti_image(nifti_image)
+        nifti_image_as_np_array: np.array = normalise_nifti_image(nifti_image)
```

Comment:
It gets harder to understand because now in "normalise", you actually do two things: you transform from nifti to numpy, _AND_ you normalise. Maybe you can change the name to "extract_np_array_from_nifti_image_and_normalise"